src/main/scala/Domain.scala

```diff
@@ -1,16 +1,15 @@
 package simpleCQRS
 
+class InventoryItem (private var _id : UUID,
+                     private var name: String)
```

Comment:
Such an alignment of fields (most probably done by IDEA) is unfriendly to refactoring. Try to rename InventoryItem to something shorter/longer and you will see.